src/main/java/racinggame/domain/Validator.java

```diff
@@ -1,29 +1,59 @@
 package racinggame.domain;
 
+import static racinggame.exception.ExceptionBaseMessage.INPUT_BLANK_MESSAGE;
+import static racinggame.exception.ExceptionBaseMessage.LACK_OF_PLAYER_MESSAGE;
+import static racinggame.exception.ExceptionBaseMessage.NAME_LENGTH_OVER_MESSAGE;
+
+import java.util.List;
+import racinggame.exception.InputBlankException;
+import racinggame.exception.LackOfPlayerException;
+import racinggame.exception.NameLengthOverException;
+
 public class Validator {
 
     public static final int MINIMUM_PLAYER = 2;
     private static final int MAXIMUM_NAME_LENGTH = 5;
 
     private Validator() {}
 
-    public static void validatePossibleToStart(String[] carNames) {
-        if (carNames.length < MINIMUM_PLAYER) {
-            throw new IllegalArgumentException(String.format("참가자는 %d명 이상입니다.", MINIMUM_PLAYER));
+    public static void validatePossibleToStart(final RacingCars racingCars) {
+        if (isPossibleToStart(racingCars.get())) {
+            throw new LackOfPlayerException(
+                    String.format("%s (현재 참여인원 %d)", LACK_OF_PLAYER_MESSAGE, racingCars.get().size()));
         }
     }
 
-    public static void validateNameLength(String carName) {
+    private static boolean isPossibleToStart(final List<RacingCar> racingCars) {
+        return racingCars.size() < MINIMUM_PLAYER;
+    }
+
+    public static void validateNameLength(final String carName) {
         if (carName.length() > MAXIMUM_NAME_LENGTH) {
-            throw new IllegalArgumentException(String.format("자동차 이름의 글자 수는 %d자 이하입니다.", MAXIMUM_NAME_LENGTH));
+            throw new NameLengthOverException(
+                    String.format("%s (Input: %s)", NAME_LENGTH_OVER_MESSAGE, carName));
         }
     }
 
-    public static void isNumber(String trial) {
+    public static void isNumber(final String trial) {
         try {
             Integer.parseInt(trial);
         } catch (NumberFormatException e) {
-            throw new IllegalArgumentException("시도횟수는 숫자로 입력해주세요.");
+            throw new NumberFormatException("시도횟수는 숫자로 입력해주세요.");
+        }
+    }
+
+    public static void checkIsBlank(final String input) {
+        if (input.isEmpty() | input.matches("\\s+")) {
```

Comment:
```suggestion
        if (input.isEmpty() | input.matches(SPACE)) {
```
가독성을 위해 매직넘버를 제거 하는 것은 어떨까요?

Reply:
감사합니다!!